Source/WebCore/platform/graphics/cv/CVUtilities.mm

```diff
@@ -180,6 +180,9 @@ void setOwnershipIdentityForCVPixelBuffer(CVPixelBufferRef pixelBuffer, const Pr
 
     NSDictionary *pixelAttributes = @{ (__bridge NSString *)kCVPixelBufferIOSurfacePropertiesKey : @{ } };
 
+    if (width * height > 35651584)
```

Comment:
@youennf should I have the limit here or in `LibWebRTCCodecs::encodeFrame`?

Reply:
@jhertz-apple reported another repro which occurs in `SharedVideoFrameReader::readBuffer` (which proceeds to call `WebCore::createBlackPixelBuffer`) so perhaps leaving the check in `WebCore::createBlackPixelBuffer` is most appropriate